Dockerfile-slim

```diff
@@ -21,16 +21,13 @@ RUN npm ci --ignore-scripts
 # conditional copies and the arm64 folder is ignored on amd64 anyway
 COPY --from=esy /app/_release /grain/node_modules/esy/platform-linux-arm64
 # Manually run esy's postinstall script
-RUN cd node_modules/esy && npm run postinstall
-
-# Necessary because we disabled scripts during the original install
-RUN npm run prepare
-
-# Build the compiler and CLI
-RUN npm run compiler build
-
-# Remove build files
-RUN rm -rf compiler/_esy
+RUN cd node_modules/esy && npm run postinstall \
+    # Necessary because we disabled scripts during the original install
+    && npm run prepare \
+    # Build the compiler and CLI
+    && npm run compiler build \
+    # Remove build files
+    && rm -rf compiler/_esy
```

Comment:
Similar idea here, esy's postinstall script, the prepare script, and the build script should all be their own RUN statements, but combining the build and the removal of the build files into a single step is an improvement :)